slave.mk

```diff
@@ -1042,6 +1044,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 
 		DEBUG_IMG="$(INSTALL_DEBUG_TOOLS)" \
 		DEBUG_SRC_ARCHIVE_FILE="$(DBG_SRC_ARCHIVE_FILE)" \
+		BUILD_TARGET="$@" \
```

Comment:
i am not sure why this change is needed, can you explain?

Reply:
Removed

Reply:
See the reply from the same question from you to Bill last year, removed this for now =>

@lguohan lguohan on 29 Feb 2020 Contributor
I do not this BUILD_TARGET used anyway. is there something else missing?
 
@wbschwar wbschwar on 24 Sep 2020 Author
We pass the BUILD_TARGET to the docker build slave so we can support the build target sonic-broadcom-pde.bin to differenciate it from the normal build. Please let me know if there is a better way to handle this.

Reply:
Looks like the build is fail now. Remove BUILD_TARGET="$@" \ should not affect the build